backend/infrahub/core/schema_manager.py

```diff
@@ -904,8 +904,11 @@ def validate_menu_placements(self) -> None:
                 try:
                     placement_node = self.get(name=node.menu_placement, duplicate=False)
                 except SchemaNotFoundError:
-                    raise ValueError(f"{node.kind}: {node.menu_placement} is not a valid menu placement") from None
-
+                    raise SchemaNotFoundError(
+                        branch_name=self.name,
+                        identifier=node.menu_placement,
+                        message=f"{node.kind} refers an unexisting menu placement node: {node.menu_placement}.",
```

Comment:
I'd say `f"{node.kind} refers to an invalid menu placement node: {node.menu_placement}."`